tests/FlatBuffers.Test.Nim/tests/moredefaults/test.nim

```diff
@@ -0,0 +1,28 @@
+discard """
```

Comment:
could you put the tests in `tests/nim/....` ? This applies all the rest of the files.